doc/man3/EVP_PKEY_CTX_new.pod

```diff
@@ -50,7 +50,10 @@ for example during parameter generation or key generation for some
 algorithms.
 
 EVP_PKEY_CTX_dup() duplicates the context I<ctx>. It is not supported for a
-keygen operation.
+keygen operation.  However, as of OpenSSL 3.6, it is possible to duplicate a
```

Comment:
I think this is a bug so should be backported to stable branches so there is no need to mention 3.6 explicitly.

Reply:
Adding backport labels.